.github/workflows/deploy-dev.yml

```diff
@@ -115,6 +125,29 @@ jobs:
       WFNEWS_BASEURL: https://wfnews-api.dev.bcwildfireservices.com/
       WFNEWS_QUEUESIZE: 1500
 
+      #WFONE-PUSH-NOTIFICATION-API VARIABLES
+      WFONE_PUSH_NOTIFICATION_DATASOURCE_URL: "jdbc:postgresql://wfone-public-mobile-db-nonprod.caw9xhtwiyin.ca-central-1.rds.amazonaws.com/wfone_int_pm"
+      WFONE_PUSH_NOTIFICATION_DATASOURCE_USERNAME: "proxy_wf1_notification_rest_int"
+      WFONE_PM_SQS_S3_BUCKET_NAME: "wfone-public-mobile-monitor-queue-bucket-int"
+      WFONE_PUSH_NOTIFICATION_MAX_CONNECTIONS: 25
+      WFONE_PUSH_NOTIFICATION_EXPIRE_HOURS: 24
+      WFONE_SQS_QUEUE_NOTIFICATION_URL: https://sqs.ca-central-1.amazonaws.com/460053263286/wfone-public-mobile-int-notification-api-queue
```

Comment:
This is an existing, Vivid-owned SQS queue - we'll need to create new queues owned by the ministry AWS account rather than us. There should be examples in wfone-notification-monitors-api

Reply:
@vivid-cpreston / @ssylver93 This queue should actually already exist from the setup work I did earlier. Take a look at the terraform scripts to verify